tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -15,6 +15,8 @@ public static DataFrame MakeDataFrameWithTwoColumns(int length)
         {
             BaseColumn dataFrameColumn1 = new PrimitiveColumn<int>("Int1", Enumerable.Range(0, length).Select(x => x));
             BaseColumn dataFrameColumn2 = new PrimitiveColumn<int>("Int2", Enumerable.Range(10, length).Select(x => x));
+            dataFrameColumn1[length/2] = null;
```

Comment:
Nit: spacing

```suggestion
            dataFrameColumn1[length / 2] = null;
```
Same on next line.

Reply:
Damn, I wonder why the .editorconfig did not find this